src/types/Program.ts

```diff
@@ -0,0 +1,6 @@
+export type Program =
+  | "Healthy Habits For The Long Haul"
+  | "Diabetes Prevention"
+  | "Rigs without Cigs"
```

Comment:
```suggestion
  | "Rigs Without Cigs"
```

Reply:
It actually is with the lowercase w but I can get behind making it all caps because it looks nicer lol